e2e-tests/gef/cypress/e2e/amendments/portal-amendments-disabled/make-a-change-button.spec.js

```diff
@@ -0,0 +1,66 @@
+import { FACILITY_TYPE } from '@ukef/dtfs2-common';
+import relative from '../../relativeURL';
+import CONSTANTS from '../../../fixtures/constants';
+import { MOCK_APPLICATION_AIN } from '../../../fixtures/mocks/mock-deals';
+import { BANK1_MAKER1, BANK1_CHECKER1_WITH_MOCK_ID } from '../../../../../e2e-fixtures/portal-users.fixture';
+import { multipleMockGefFacilities } from '../../../../../e2e-fixtures/mock-gef-facilities';
+import applicationPreview from '../../pages/application-preview';
+
+const { unissuedCashFacility, unissuedContingentFacility, anIssuedContingentFacility, issuedCashFacility } = multipleMockGefFacilities();
+
+let dealId;
+let token;
+
+let issuedCashFacilityId;
+let issuedContingentFacilityId;
+let unissuedCashFacilityId;
+let unissuedContingentFacilityId;
+
+context('Make a change button - FF_PORTAL_FACILITY_AMENDMENTS_ENABLED feature flag disabled', () => {
```

Comment:
Can we add `Amendments -` to the beginning of the title please

Reply:
Yep, now added